ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js

```diff
@@ -61,13 +61,14 @@ export default function RecoveryPhrase({ secretRecoveryPhrase }) {
 
   useEffect(() => {
     if (!secretRecoveryPhrase) {
-      history.replace(
-        `${ONBOARDING_REVEAL_SRP_ROUTE}/${
+      navigate(
```

Comment:
Removed the hardcoded `/` because it created trailing slashes that don't match the route definitions in React Router v6's strict path matching system.